src/mscorlib/src/System/PinnableBufferCacheEventSource.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+namespace System
+{
+    [EventSource(Name = "Microsoft-DotNETRuntime-PinnableBufferCache-System")]
+    internal sealed class PinnableBufferCacheEventSource : EventSource
```

Comment:
Could you add the Guid to the attribute and also add a .ctor that calls the base with that guid as per ArrayPoolEventSource https://github.com/dotnet/coreclr/pull/16054
```csharp
private PinnableBufferCacheEventSource() : base(new Guid( ...
```
Otherwise the startup for the EventSource becomes costly https://github.com/dotnet/coreclr/issues/15954